### PR TITLE
Testing

### PR DESCRIPTION
**Pull request by DaanVermunt**
Upped test coverage, didn't change any working code.

### CODE REVIEW

**Comment by arjovanramshorst** [comment]
assertTrue(true) in levelparsertest doesn't test the code as supposed. There should be some way to assert that an exception is thrown.

**Comment by arjovanramshorst** [comment]
Other than that it looks good

**Comment by wouterz** [comment]
looks good, however testing Floor-/WallObjects update issnt really meaningful as it is an empty method <testing

File: core/src/test/java/sem/group15/bubblebobble/core/LevelParserTest.java [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+        file = mock(FileHandle.class);
+    }
+    @Test
+    public void testIOExeption(){
+        String str = "Floors, 200, 100";
+        try {
+            when(file.read()).thenReturn(new ByteArrayInputStream(str.getBytes("UTF-8")));
+            new LevelParser().parse(file);
+        }catch (IOException e){
+            assertTrue(true);
```
Comment (arjovanramshorst) [inline core/src/test/java/sem/group15/bubblebobble/core/LevelParserTest.java]: Niet echt juiste manier om te checken of een exception gethrowed wordt. Als ie niet gethrowed wordt, passed de test ook

File: core/src/test/java/sem/group15/bubblebobble/core/objects/FloatingObjectTest.java [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+    @Before
+    public void setUp(){
+        Gdx.app = mock(Application.class);
+        Gdx.input = mock(Input.class);
+        floatingObject = mock(FloatingObject.class, Mockito.CALLS_REAL_METHODS);
+        floatingObject.location = new Rectangle(0, 0, BubbleBobble.SPRITE_SIZE, BubbleBobble.SPRITE_SIZE);
+    }
+
+    @Test
+    public void testUpdate(){
```
Comment (arjovanramshorst) [inline core/src/test/java/sem/group15/bubblebobble/core/objects/FloatingObjectTest.java]: Misschien deze functie iets anders noemen: testUpdateTimeFromFiredIncreasesProperly bijvoorbeeld.

File: core/src/test/java/sem/group15/bubblebobble/core/objects/WallObjectTest.java [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+        wall = mock(WallObject.class, Mockito.CALLS_REAL_METHODS);
+        wall.location = new Rectangle(0, 0, BubbleBobble.SPRITE_SIZE, BubbleBobble.SPRITE_SIZE);
+
+    }
+
+    /**
+     * Test if wall stays in the same place if update is called.
+     */
+    @Test
+    public void testUpdate(){
```
Comment (arjovanramshorst) [inline core/src/test/java/sem/group15/bubblebobble/core/objects/WallObjectTest.java]: Hier ook: veranderen in bijvoorbeeld: testWallPositionNotChangedAfterUpdate

File: core/src/test/java/sem/group15/bubblebobble/core/objects/PlayerObjectTest.java [hunk 4]
```diff
@@ -258,4 +259,5 @@ public void testJump3() {
         assertFalse(player.canJump);
         verify(player, never()).playJumpSound();
     }
+
 }
```
Comment (MatthijsRijlaarsdam) [inline core/src/test/java/sem/group15/bubblebobble/core/objects/PlayerObjectTest.java]: watch your checkstyle! <layout

**Comment by MatthijsRijlaarsdam** [comment]
Some minor notes, checkstyle we'll fix during the final checkup. I'll merge! <testing

**Pull request merged**